src/sdk/accountspace_fs.js

```diff
@@ -468,44 +447,6 @@ class AccountSpaceFS {
     // INTERNAL FUNCTIONS //
     ////////////////////////
 
-     _get_account_config_path(name) {
-        return get_config_file_path(this.accounts_dir, name);
-     }
-
-     _get_access_keys_config_path(access_key) {
-        return get_symlink_config_file_path(this.access_keys_dir, access_key);
-     }
-
-     async _get_account_decrypted_data_optional(account_path, should_decrypt_secret_key) {
-        const data = await get_config_data(this.config_root_backend, account_path, true);
-        if (should_decrypt_secret_key) data.access_keys = await nc_mkm.decrypt_access_keys(data);
-        return data;
-     }
-
-     /**
-     * _get_account_decrypted_data_optional_if_exists will read a config file and return its content
-     * if the config file was deleted (encounter ENOENT error) - continue (returns undefined)
-     *
-     * Notes: this function is important when dealing with concurrency.
-     * When we iterate files (for example for listing them) between the time we read the entries
-     * from the directory and the time we we are trying to read the config file,
-     * a file might be deleted (by another process), and we would not want to throw this error
-     * as a part of iterating the file, therefore we continue
-     * (not throwing this error and return undefined)
```

Comment:
Could you please move this comment to the function in class `configFS`?
It it a comment that you asked me to add